config/cbridge.json

```diff
@@ -46,6 +46,7 @@
   "polygon": {
     "cBridge": "0x88DCDC47D2f83a99CF0000FDF667A468bB958a78",
     "messageBus": "0xaFDb9C40C7144022811F034EE07Ce2E110093fe6",
+    "cfUSDC": "0x0000000000000000000000000000000000000000",
```

Comment:
nit: actually, in deploy script, it gets `cfUSDC` address only when it deploys on mainnet.
     so it doesn't need to add this.

Reply:
![image](https://github.com/lifinance/contracts/assets/112938295/87e56cc9-73d0-4024-b2af-a1f3c2a38fc5)

Reply:
true, forgot about that...i've removed it from config